src/System.Management.Automation/engine/COM/ComUtil.cs

```diff
@@ -57,27 +54,9 @@ internal static string GetMethodSignatureFromFuncDesc(COM.ITypeInfo typeinfo, CO
 
             for (int i = 0; i < funcdesc.cParams; i++)
             {
-                COM.ELEMDESC ElementDescription;
-                int ElementDescriptionArrayByteOffset;
-                IntPtr ElementDescriptionPointer;
-
-                ElementDescription = new COM.ELEMDESC();
-                ElementDescriptionArrayByteOffset = i * ElementDescriptionSize;
-
-                // Disable PRefast warning for converting to int32 and converting back into intptr.
-                // Code below takes into account 32 bit vs 64 bit conversions
-#pragma warning disable 56515
-                if (IntPtr.Size == 4)
```

Comment:
How are we sure that this is not applicable?

Reply:
In the original code `IntPtr.Size == 4` is checking whether we are 32 or 64 bit.

We do not need such a check anymore as
 `IntPtr.Addition(IntPtr, Int32) operator` handles both cases.